README.md

```diff
@@ -132,7 +132,9 @@ Before you begin, ensure that you have the following installed:
 ### Build and Start Docker Containers
 This command will rebuild and start all the services using the configuration defined in docker-compose.yaml
 
-```
+```bash
+export GID=$(id -g)
+export UID=$(id -u)
```

Comment:
Can we add these variables to `.env` file?

Reply:
Technically no. Because the users need to find their `UID` and `GID`. I have added this because for stage and production, I will be mapping these with the system's user so that the application can write to directories mounted inside the container. So its own fair to match the local development environment similarly. We'll need this specifically for the media files.

We can add a `Makefile` that takes care of this.

Reply:
Oh okay, got it. Makes sense.